Cargo.toml

```diff
@@ -18,6 +18,9 @@ is-it-maintained-issue-resolution = { repository = "BurntSushi/stdsimd" }
 is-it-maintained-open-issues = { repository = "BurntSushi/stdsimd" }
 maintenance = { status = "experimental" }
 
+[dependencies]
+coresimd = { version = "0.0.3", path = "coresimd/" }
```

Comment:
I've set the first version of `coresimd` to `0.0.3` to match that of `stdsimd`, we should release these together.